src/libs/Navigation/shouldSetScreenBlurred/index.tsx

```diff
@@ -0,0 +1,13 @@
+/**
+ * @param navigationIndex
+ *
+ * Decides whether to set screen to blurred state.
+ *
+ * Allow freezing the first screen and more in the stack only on
+ * web and desktop platforms. The reason is that in the case of
+ * LHN, we have FlashList rendering in the back while we are on
+ * Settings screen.
```

Comment:
Is this assuming then that web/desktop is wide layout and native is narrow layout? 

Its not a best practice and we should avoid that if that is the case as we are working on allowing wide layout on native ipad too and then this would not work that well. Not a blocker though